Assets/Scripts/EnemyController.cs

```diff
@@ -1,30 +1,39 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
 public class EnemyController : MonoBehaviour
 {
     [SerializeField]
     private float movementSpeed;
     [SerializeField]
-    private int maxHealth = 10; 
+    private int maxHealth = 10;
     [SerializeField]
     private int currentHealth;
+    [SerializeField]
+    private float fireRate = 1f;
+    [SerializeField]
+    private GameObject proj;
+    [SerializeField]
+    private Transform playerPos;
```

Comment:
Suggested naming changes:
- `proj` -> `projectile`
  - It is not clear what `proj` means just by looking at the instance variables in this script. I'm not sure if `ProjController` should be changed to `ProjectileController`, but I think `ProjectileController` would be better.
- `playerPos` -> `playerTransform`
  - Position is just a property of the transform, so `playerTransform` is more accurate. (I think `player` would also be fine.)